scraping_utils.py

```diff
@@ -0,0 +1,305 @@
+import sys
```

Comment:
We started off pretty good with documentation, but this new file is lacking docstrings which is something that makes it much easier to understand the code from the point of view of other programmers, especially if people other than you and I want to contribute we should try and keep the code as well commented as possible to make it easier for new entrants to join the team.

Reply:
I will work on this and let you know when I've gotten it done!